nixos/modules/services/web-apps/peertube.nix

```diff
@@ -282,6 +294,18 @@ in {
             prevent this.
           '';
       }
+      { assertion = cfg.secrets.secretsFile != null;
+          message = ''
+            <option>services.peertube.secrets.secretsFile</option> needs to be set.
+          '';
+      }
```

Comment:
Maybe just don't set any default for the setting

Reply:
Now without this parameter the peertube service will not start:
```
peertube-start[116937]: [stream.elven:443] 2022-12-13 23:28:21.614 debug: Checking NodeJS version v16.18.1.
peertube-start[116937]: [stream.elven:443] 2022-12-13 23:28:24.609 info: Using following configuration file hierarchy: /var/lib/peertube/config/default.yaml -> /var/lib/peertube/config/production.json -> /var/lib/peertube/config/local.yaml -> /var/lib/peertube/config/local-production.json.
peertube-start[116937]: [stream.elven:443] 2022-12-13 23:28:24.610 error: Config error. {
peertube-start[116937]:   "err": {
peertube-start[116937]:     "stack": "Error: secrets.peertube is missing in config. Generate one using `openssl rand -hex 32`\n    at checkSecretsConfig (/nix/store/bcz7xn28h58scl6744idj8xhq3hm9cxg-peertube-5.0.0/dist/server/initializerschecker-after-init.js:88:15)\n    at checkConfig (/nix/store/bcz7xn28h58scl6744idj8xhq3hm9cxg-peertube-5.0.0/dist/server/initializers/checker-after-init.js:37:5)\n    at Object.<anonymous> (/nix/store/bcz7xn28h58scl6744idj8xhq3hm9cxg-peertube-5.0.0/dist/server.js:39:42)\n    at Module._compile (node:internal/modules/cjs/loader:1155:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)\n    at Module.load (node:internal/modules/cjs/loader:1033:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:868:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:22:47",
peertube-start[116937]:     "message": "secrets.peertube is missing in config. Generate one using `openssl rand -hex 32`"
peertube-start[116937]:   }
peertube-start[116937]: }
```